zio-http/src/main/scala/zhttp/service/Server.scala

```diff
@@ -146,6 +147,7 @@ object Server {
     consolidateFlush: Boolean = false,
     flowControl: Boolean = true,
     channelInitializer: ChannelPipeline => Unit = null,
+    objectAggregator: Int = -1,
   )
```

Comment:
```suggestion
  )
  
  def useAggregator: Boolean = objectAggregator > 0
```

And use this method everywhere.

Reply:
done